### PR TITLE
[docs] Move react-native-unimodules/expo-updates installation instructions from diffs in README to dedicate docs

### PR DESCRIPTION
**Pull request by brentvatne**
# Why

They are currently very hard to follow in the READMEs, we have the technology to make them better so let's use it.

# How

Use the same diff tool as https://react-native-community.github.io/upgrade-helper/ but much more barebones.

The implementation here isn't perfect but I think it's a lot better and something we can continue to iterate on.

# Test Plan

Check it out, look at the guides

# Screenshots

Here are some random screenshots of the new docs pages to give you an idea.

Screen Shot 2020-07-08 at 7 57 34 PM
Screen Shot 2020-07-08 at 7 57 41 PM
Screen Shot 2020-07-08 at 7 57 38 PM

Screen Shot 2020-07-08 at 7 57 47 PM
Screen Shot 2020-07-08 at 7 57 50 PM
Screen Shot 2020-07-08 at 7 57 52 PM
Screen Shot 2020-07-08 at 7 57 54 PM

### CODE REVIEW

**Review by tsapeta — APPROVED** [review]
Yeah, looks much better. Would be even better if that could also support syntax highlighting 🙏

**Comment by brentvatne** [comment]
> Yeah, looks much better. Would be even better if that could also support syntax highlighting 🙏

this would be nice! i looked into it briefly but it seems non-trivial to build with react-diff-view so i'll defer this for now.

File: docs/static/diffs/expo-updates-android.diff [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+       android:roundIcon="@mipmap/ic_launcher_round"
+       android:allowBackup="false"
+       android:theme="@style/AppTheme">
+-      <activity
+-        android:name=".MainActivity"
++      <meta-data android:name="expo.modules.updates.EXPO_UPDATE_URL" android:value="https://exp.host/@my-expo-username/my-app" />
++      <meta-data android:name="expo.modules.updates.EXPO_SDK_VERSION" android:value="38.0.0" />
++
++  <activity
++    android:name=".MainActivity"
```
Comment (esamelson) [inline docs/static/diffs/expo-updates-android.diff:33]: this diff is a little weird, only the lines on 29 and 30 need to be added, the other changes in the diff are not meaningful

**Review by esamelson — APPROVED** [review]
Looks great, thanks @brentvatne for not only making the experience better but also reorganizing the text/copy to be clearer!

File: docs/static/diffs/expo-updates-android.diff (hunk 1 above)
Comment (brentvatne) [reply docs/static/diffs/expo-updates-android.diff]: good point, manually changed this
